<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dynamic VoIP UI (Qt Conversion)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayoutRoot">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="headerFrame">
      <property name="frameShape">
       <enum>QFrame::Shape::NoFrame</enum>
      </property>
      <layout class="QHBoxLayout" name="headerLayout">
       <property name="leftMargin">
        <number>16</number>
       </property>
       <property name="topMargin">
        <number>12</number>
       </property>
       <property name="rightMargin">
        <number>16</number>
       </property>
       <property name="bottomMargin">
        <number>12</number>
       </property>
       <item>
        <widget class="QLabel" name="lblServerDot">
         <property name="minimumSize">
          <size>
           <width>12</width>
           <height>12</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>12</width>
           <height>12</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblServerState">
         <property name="text">
          <string>Connecting...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblLatencyTitle">
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblLatencyValue">
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacerHeader">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="lblUsername">
         <property name="text">
          <string>You</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chkMicActive">
         <property name="text">
          <string>Mic Enabled</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblMuteState">
         <property name="text">
          <string>Active</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="mainSplitter">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <widget class="QWidget" name="leftPanel">
       <layout class="QVBoxLayout" name="leftLayout">
        <property name="leftMargin">
         <number>12</number>
        </property>
        <property name="topMargin">
         <number>12</number>
        </property>
        <property name="rightMargin">
         <number>12</number>
        </property>
        <property name="bottomMargin">
         <number>12</number>
        </property>
        <item>
         <widget class="QLineEdit" name="editSearch">
         <property name="placeholderText">
           <string>Search by name, ID, or status...</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="statsLayout">
          <item>
           <widget class="QLabel" name="lblOnlineCount">
            <property name="text">
             <string>0 online</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblInCallCount">
           <property name="text">
             <string>0 talking</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblOfflineCount">
            <property name="text">
             <string>0 offline</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListWidget" name="listClients"/>
        </item>
        <item>
         <widget class="QPushButton" name="btnGroupCall">
         <property name="text">
           <string>Select client(s) to talk</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnBroadcastAll">
          <property name="text">
           <string>Broadcast to All</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="rightPanel">
       <layout class="QVBoxLayout" name="rightLayout">
        <property name="leftMargin">
         <number>12</number>
        </property>
        <property name="topMargin">
         <number>12</number>
        </property>
        <property name="rightMargin">
         <number>12</number>
        </property>
        <property name="bottomMargin">
         <number>12</number>
        </property>
        <item>
         <widget class="QStackedWidget" name="callStack">
          <widget class="QWidget" name="pageIdle">
           <layout class="QVBoxLayout" name="idleLayout">
            <item>
             <spacer name="verticalSpacerIdleTop">
              <property name="orientation">
               <enum>Qt::Orientation::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>120</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="lblNoCallTitle">
              <property name="text">
               <string>No Active Call</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lblNoCallSubtitle">
              <property name="text">
               <string>Start a broadcast or select clients from the sidebar</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnStageBroadcast">
              <property name="text">
               <string>Start Broadcast</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacerIdleBottom">
              <property name="orientation">
               <enum>Qt::Orientation::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>120</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="pageActive">
           <layout class="QVBoxLayout" name="activeLayout">
            <item>
             <widget class="QLabel" name="lblCallType">
              <property name="text">
               <string>CALL</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lblParticipantCount">
              <property name="text">
               <string>0 participant(s)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lblLiveBroadcast">
              <property name="text">
               <string>LIVE BROADCAST</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QListWidget" name="listParticipants"/>
            </item>
            <item>
             <widget class="QPushButton" name="btnPushToTalk">
              <property name="text">
               <string>Hold to Broadcast</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lblPushHint">
              <property name="text">
               <string>Press and hold to push-to-talk</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="bottomFrame">
      <property name="frameShape">
       <enum>QFrame::Shape::NoFrame</enum>
      </property>
      <layout class="QHBoxLayout" name="bottomLayout">
       <property name="leftMargin">
        <number>16</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>16</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="visualizerLayout">
         <item>
          <widget class="QProgressBar" name="bar1"/>
         </item>
         <item>
          <widget class="QProgressBar" name="bar2"/>
         </item>
         <item>
          <widget class="QProgressBar" name="bar3"/>
         </item>
         <item>
          <widget class="QProgressBar" name="bar4"/>
         </item>
         <item>
          <widget class="QProgressBar" name="bar5"/>
         </item>
         <item>
          <widget class="QProgressBar" name="bar6"/>
         </item>
         <item>
          <widget class="QProgressBar" name="bar7"/>
         </item>
         <item>
          <widget class="QProgressBar" name="bar8"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="lblMicActivity">
         <property name="text">
          <string>Mic Idle</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblMicLabel">
         <property name="text">
          <string>Mic</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="sliderMic">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblMicValue">
         <property name="text">
          <string>75%</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblSpeakerLabel">
         <property name="text">
          <string>Speaker</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="sliderSpeaker">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblSpeakerValue">
         <property name="text">
          <string>80%</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnEndCall">
         <property name="text">
          <string>End Call</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
